pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -849,6 +953,10 @@ protected Map<String, Object> buildPayloadMap() {
         return payload;
     }
 
+    private boolean isLiveActivityPayload() {
+        return event != null;
+    }
```

Comment:
Trivial: since this is only used in one method and since it's not being exposed for tests, I'd be tempted to just inline this in `buildPayloadMap` as:

```
final boolean isLiveActivityPayload = event != null;
```